pkg/testrunner/scenario.go

```diff
@@ -133,25 +134,21 @@ func runScenario(t *testing.T, s *scenarioT) bool {
 }
 
 func runTestCase(t *testing.T, tc scaseT) bool {
-
-	// apply policy
```

Comment:
curious why the comments were removed ?

Reply:
Comments that state "what" the code is doing add clutter to code. A good guideline is to prefer to use "why" comments and avoid "what" comments.

Overall, its best to use comments judiciously and always try and replace comments with better naming and code structure. For example, would this method be better named as "convertPolicy"?

Reply:
true, added an issue to re-work the test-runner framework to make it more readable.
https://github.com/nirmata/kyverno/issues/454